src/material/stepper/stepper.md

```diff
@@ -222,3 +222,19 @@ is given `role="tab"`, and the content that can be expanded upon selection is gi
 step content is automatically set based on step selection change.
 
 The stepper and each step should be given a meaningful label via `aria-label` or `aria-labelledby`.
+
+When a stepper is used on a smaller screen, verical steppers are preferred over
+horizontal steppers because the text in the header will be more visible.
+
+#### Forms
+Steppers often have forms inside of it. It's best practice for form fields to
+have `<mat-error>` for fields that can be incorrect and throw an error.
```

Comment:
nit: I'm not sure that "throw an error" is the correct terminology here since it sounds like there was an actual runtime error, whereas these are validation errors.